### DISCUSSION TITLE
Runtime error. klassen er ikke registreret

### DISCUSSION THREAD

**Discussion by WinterBoyTV** [Eduap-com/WordMat#67] [Support] [answered]
Efter opdatering af programmet til version 1.24.2, kan jeg ikke længere lave nogen udregninger. Jeg har forsøgt at geninstallere det gentagne gange - også mens jeg har haft slået mit antivirus fra. Fejlen kan ses i billederne.
![image](https://user-images.githubusercontent.com/105121086/167262633-b4a9d346-be42-41d5-a700-2894ce77d4a5.png)
![image](https://user-images.githubusercontent.com/105121086/167262673-9bb89163-bdca-4132-9e6d-ecdc6b962e70.png)

**Comment by Eduap-com** [comment]
Det tyder på at installationen er blevet forstyrret. Det er formentlig ikke nok at deaktivere antivirus. Det skal helt afinstalleres. 
Som alternativ kan du også vælge at bruge GeoGebra som CAS-Motor i indstillingerne. Det vil formentlig virke.

**Reply by Eduap-com** [reply]
Jeg har nu set en computer med dette problem, uden at kunne identificere problemet. Mit bedste bud er at et eller andet program på computeren forstyrrer.

**Comment by WinterBoyTV** [comment]
Så jeg skal helt afinstallere mit antivirus og så forsøge at downloade WordMat igen?
Forresten så gjorde det ingen forskel at skifte til GeoGebra i CAS-Motor indstillingen. Det gav den samme runtime error. Dog virkede GeoGebra browser, men det er til gengæld ikke en indstilling, jeg ønsker at bruge, da den jo laver udregning mm. i browseren frem for i dokumentet.

**Comment by Eduap-com** [comment]
'GeoGebra browser' er en sidste backup, hvis forskellige problemer umuliggør de to andre CAS-muligheder.
Du må meget gerne skrive hvilken antivirus du bruger, og om det virkede med helt at afinstallere programmet.

**Comment by MatrixModMenuTeam** [comment]
Jeg døjer med samme problem, jeg kører helt normalt windows defender. Har prøvet at deaktivere det, og downloade word mat igen. Men kan ikke få det til at virke, er der fundet en løsning?

**Comment by FreshmeatDK** [comment]
Jeg har samme problem. Windows 10 Home 64 bit, 21H2 ver 19044.2006 på Surface Pro 4. Office 365, Word 2209 Build 15629.20156 click and run. Har installeret med Windows Defender slået fra, har ikke andre virusskjold. Problemet består når jeg ruller tilbage til 1.24, og 1.22 bliver ikke længere genkendt af Word, når det installeres.

**Comment by Eduap-com** [comment, marked as answer]
Jeg har nu mødt en computer med dette problem.
Det løste problemet at foretage en en online-reparation af Office, og dernæst geninstallere Office.

Det gøres via ’tilføj eller fjern programmer’. Find Office på listen. Den kan fx hedde ‘Microsoft 365 – da-dk’ eller ‘Microsoft 365 App for enterprise – da-dk’.
Højreklik og vælg rediger. Dernæst online reparation. Efter reparationen af Office, skal WordMat genistalleres.

Problemet kan formentlig skyldes flere ting
Det kan også skyldes at WordMat ikke er installeret korrekt, hvis nogle elementer af installationen er blevet blokeret af antivirus eller lignende. Fjern alt antivirus inden der installeres igen. Det er sjældent nok at deaktivere antivirus.
Det har også skyldes Windows Defender. Man kan prøve at tilføje mappen ‘Programmer (x86)/WordMat’ til at være undtaget Defender.
Det kan også være et midlertidigt problem, som man kan afvente at Microsoft får løst.
